CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2022-07-24
+
```

Comment:
should we add estimated time for rollout completion?

Reply:
No, that's what the tracker is for